use_cases/community-manager.md

```diff
@@ -0,0 +1,215 @@
+
+## Vocabulary
+* Community Manager: takes a variety of perspectives, depending on where their communities are in the lifecycle of growth, maturity and decline. 
+
+* Community: A collection of interconnected repositories. Sometimes referred to as an "ecosystem" if it scales largely enough. 
+
+* Community Manager Metrics: This is an evolving use case of what we are learning from community managers, some of whom we are working with on live experiments with a CHAOSS project prototyping software tool called Augur (http://www.github.com/CHAOSS/augur). At this point we are paying particular focus to how community managers consume metrics and how the presentation of open source software health and sustainability metrics could make them more and in some cases less useful for doing their jobs. 
+
+* Others as defined in [First Code Contribution](./first-code-contributions.md) use case. 
+
+## Primary actor
+
+Person responsible for managing the community of repositories. 
+
+## Stakeholders
+
+* Community manager for the project, for tracking how the onboarding process is performing.
+
+* Project community in general, for self-awareness about how the onboarding
+process is performing.
+
+* Prospective contributors, to make decisions on whether to
+invest their time and effort in joining a community,
+or in general, produce contributions for the project.
+
+## Preconditions
+
+* The community manager should have an interest in a broad perspective on many projects. 
+* The community manager should have a larger than 1 set of repositories they are interested in tracking collectively.  
+
+## Questions
+
+* Embedded in Metrics Definition
+
+## Metrics
+
+\[Work in progress\]
+
+Right now, based on Augur prototypes and follow up discussions so far,
+we have the following observations that will inform our work both the
+the \"Growth Maturity and Decline\" working group and in Augur
+Development. There are a few things we have learned from prototyping
+Augur with community managers. These features in Augur are particularly
+valued:
+
+1.  Allowing comparisons with projects within a defined universe is
+    essential
+
+2.  Allow community managers to add and remove repositories that they
+    monitor from their repertoires periodically.
+
+3.  Downloadable graphics
+
+4.  Downloadable data (.csv or .json)
+
+5.  Availability of a \"Metrics API\", limiting the amount of software
+    infrastructure the CM needs to maintain for themselves. This is more
+    valued by program managers overseeing larger portfolios right now,
+    but we think has potential to grow as awareness of the relatively
+    light weight of this approach becomes more apparent. By apparent, we
+    really mean \"easy to use and understand\"; right now it is for a
+    programmer, but less so for a community manager without this
+    background or current interest.
+
+Date Summarized Comparison Metrics
+==================================
```

Comment:
lines 76-77 (and 121-122): changes Markdown formatting from using # for titles to underlining them.

Reply:
Fixed.